Ical.Net/Evaluation/Evaluator.cs

```diff
@@ -13,37 +13,16 @@ namespace Ical.Net.Evaluation;
 
 public abstract class Evaluator : IEvaluator
 {
-    private DateTime _mEvaluationStartBounds = DateTime.MaxValue;
-    private DateTime _mEvaluationEndBounds = DateTime.MinValue;
```

Comment:
Just a sidenote: Initializing with `DateTime.MinValue` in ical.net is a potential risk because we can't create a `DateTimeOffset` with it.

Reply:
Absolutely agree. So another benefit of them being gone. There are quite many of those magic values throughout the lib. Should implement `Nullable` in most of those places.